grid/recruitment/update-candidate/profile.supr

```diff
@@ -0,0 +1,397 @@
+  """
+  Bucket
+  Different parts of API may have different rate limits.
+  Bucket identifies to which part of API the rate limits apply.
+  """
+  bucket string
+  """
+  Total requests
+  Total requests available in the time window.
+  """
+  totalRequests number
+  """
+  Remaining requests
+  Remaining requests available in the time window.
+  """
+  remainingRequests number
+  """
+  Remaining requests available in percents
+  Remaining requests available in the time window in percents.
+  """
+  remainingRequestsPercentage number
+  """
+  Reset timestamp
+  Timestamp when the rate limits will reset (in  Unix Timestamp format). Available only for providers without rolling rate limit window.
+  """
+  resetTimestamp number
+}
+
+model ErrorCode enum {
+  """
+  Candidate not found
+  The candidate ID does not exist.
+  """
+  CandidateNotFound
+  """
+  Wrong integration parameter
+  The integration parameter value has not been accepted.
+  """
+  WrongIntegrationParameter
+  """
+  CV file name required
+  Missing cv.fileName input. CV file name is required when cv.data input field is defined.
+  """
+  CVFileNameRequired
+  """
+  CV MIME type not supported
+  CV MIME type is not supported by the provider. To get supported MIME types run UpdateCandidateFeatures use case.
+  """
+  CVMIMETypeNotSupported
+  """
+  Rate limit reached
+  Returned when a request cannot be served due to the user or app reached max requests quota.
+  """
+  RateLimitReached
+  """
+  Unauthenticated
+  There was a problem authenticating your request. Check that you are passing valid credentials.
+  """
+  Unauthenticated
+  """
+  Unauthorized
+  The request is understood, but it has been refused or access is not allowed.
+  """
+  Unauthorized
+  """
+  Unknown error
+  Returned when an unexpected error occured.
+  """
+  UnknownError
+}
+
+"""
+Get Update Candidate Features
+Get features of UpdateCandidate use case supported by the provider.
+"""
+usecase UpdateCandidateFeatures {
+  result {
+    """
+    CV MIME Types
+    List of CV MIME types accepted by the provider. MIME types are defined in RFC2045 and registered by https://www.iana.org/.
+    """
+    cvMIMETypes! [string!]!
+    """
+    CV Upload Methods
+    List of supported CV upload methods.
+    """
+    cvUploadMethods! [
+      enum {
+        """
+        URL
+        The cv input contains url field with publicly accessible URL of a CV file.
+        """
+        url = 'url'
+        """
+        File
+        The cv input contains data field with CV file and fileName field with the file name.
+        """
+        file = 'file'
+      }!
+    ]!
+ }
+
+  example Successful {
+    result {
+      cvMIMETypes = ['application/pdf', 'text/rtf']
+      cvUploadMethods = ['url', 'file']
+    }
+  }
+}
+
+
+"""
+Education
+Object representing educational experience of a candidate.
+"""
+model Education {
+  """
+  Id
+  Unique identifier of candidate's education experience.
+  """
+  id string
```

Comment:
This is the only "id" in the document that does not follow the convention of "<something>Id" ...